apps/frontend/src/employee/pages/EmployeeInsightCreationPage.tsx

```diff
@@ -66,43 +77,54 @@ export const EmployeeInsightCreationPage = (): ReactElement => {
                     </Heading>
-                    <Label>Valgte kandidater: {citizens ? `${candidates.length}/${citizens.length}` : '0/0'}</Label>
-                </div>
-                <div>
-                    {citizens?.map((citizen, index) => {
-                        return (
-                            <CandidatePicker
-                                key={index}
-                                citizen={citizen}
-                                index={index}
-                                insight={insight}
-                                candidates={candidates}
-                                setCandidates={setCandidates}
-                            />
-                        );
-                    }) || <APIHandler error={error} loading={loading} />}
-                    {candidateValidationErrors.candidates && (
-                        <ErrorList errorMessages={[...candidateValidationErrors.candidates]} />
-                    )}
-                </div>
-            </Panel>
-        </>
+                    <EmployeeInsightConfiguration
+                        insight={insight}
+                        setInsight={setInsight}
+                        validationErrors={insightValidationErrors}
+                    />
+                </Panel>
+                <Panel>
+                    <div className={style.candidatesHeader}>
+                        <Heading level={'2'} size="xlarge" spacing>
+                            Kandidater
+                        </Heading>
+                        <Label>Valgte kandidater: {citizens ? `${candidates.length}/${citizens.length}` : '0/0'}</Label>
+                    </div>
+                    <div>
+                        {citizens?.map((citizen, index) => {
+                            return (
+                                <CandidatePicker
+                                    key={index}
+                                    citizen={citizen}
+                                    index={index}
+                                    insight={insight}
+                                    candidates={candidates}
+                                    setCandidates={setCandidates}
+                                />
+                            );
+                        }) || <APIHandler error={error} loading={loading} />}
+                        {candidateValidationErrors.candidates && (
+                            <ErrorList errorMessages={[...candidateValidationErrors.candidates]} />
+                        )}
+                    </div>
+                </Panel>
+                <Panel>
+                    <div className={style.submit}>
+                        <Button onClick={handleSubmit}>Opprett</Button>
+                    </div>
+                </Panel>
+            </div>
+            <div>
+                <Panel>
+                    <InsightConsentForm insight={insight} />
+                </Panel>
+            </div>
+        </div>
```

Comment:
I think this should be split into multiple components

Reply:
I do agree that there probably should be two more containers here, but doing so will create a lot of prop drilling. I am hesitant on doing more changes here, before we properly figure out whether insight works should be editable or not. 

Issue on this topic: #123